src/view/fileChangeModel.ts

```diff
@@ -134,19 +134,24 @@ export class InMemFileChangeModel extends FileChangeModel {
 		const parentFileName = change.status === GitChangeType.RENAME ? change.previousFileName! : change.fileName;
 		const filePath = vscode.Uri.file(resolvePath(folderRepositoryManager.repository.rootUri, change.fileName));
 		const parentPath = vscode.Uri.file(resolvePath(folderRepositoryManager.repository.rootUri, parentFileName));
-		this._filePath = isCurrentPR ? ((change.status === GitChangeType.DELETE)
-			? toReviewUri(filePath, undefined, undefined, '', false, { base: false }, folderRepositoryManager.repository.rootUri)
-			: filePath)

```

Comment:
We need to use `file` for the active PR because we will end up with duplicate comments (one for the `file` scheme and one for the `review` scheme) if we use the `review` scheme. However, `file` is also clearly not right as the open folder doesn't always have the `file` scheme. Instead, we should use the same scheme as the currently open folder. The way I've done that in the past is to use the repository's root URI.